src/test/java/analyzer/AnalyzerIntegrationTest.java

```diff
@@ -97,4 +97,20 @@ public void twofer(String scenario) throws IOException {
 
         Approvals.verify(serialize(output.analysis()), Approvals.NAMES.withParameters(scenario));
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "ExemplarSolution",
+            "UsingForLoop",
+            "UsingIfStatement",
+            "UsingTernary",
+            "UsingWhileLoop",
+    })
+    void needforspeed(String scenario) throws IOException {
+        var path = Path.of("needforspeed", scenario + ".java");
```

Comment:
Can you change the path of the scenarios to use the slug instead (so `need-for-speed`)?

Reply:
Done!